otherlibs/stdune/src/stdune.ml

```diff
@@ -1,5 +1,4 @@
-[@@@alert unstable "The API of this library is not stable and may change without notice."]
-[@@@alert "-unstable"]
+
```

Comment:
This diff is not needed. You can test this change with 5.1 or even 4.14 to avoid the bug in OCaml.